projects/novo-elements/src/elements/data-table/interfaces.ts

```diff
@@ -12,6 +12,8 @@ export interface IDataTablePreferences {
   savedSearchId?: number;
   savedSearchName?: string;
   appliedSearchType?: AppliedSearchType;
+  hasUnsavedChanges?: boolean;
```

Comment:
I'm not sure I see these being used anywhere.

Reply:
used in latest BH-87602 `novo` changes